tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3728,7 +3733,11 @@ def runTest(self):
 
         self.sai_thrift_port_tx_disable(self.dst_client, asic_type, [dst_port_id], disable_port_by_block_queue=False)
 
-        send_packet(self, src_port_id, pkt, pkts_num_leak_out)
+        if 'cisco-8000' in asic_type:
+            queue = 1  # Dscp 0 maps to queue 1
```

Comment:
can get correct queue number befor test and pass it as parameter rather than hardcode here?

Reply:
Since #14232 has been merged now, I've removed the fill_leakout code altogether since it's already included there.